framework/contracts/native/module-factory/src/commands.rs

```diff
@@ -121,7 +127,7 @@ pub fn execute_create_modules(
                         module_addr: addr.to_string(),
                         module: new_module,
                     },
-                    vec![],
+                    new_module_init_funds,
```

Comment:
We send funds with the `register_module` action? Imo init funds should only be possible on apps and standalone, not adapters (because they are already instantiated) thoughts?

Reply:
Thank you! Fixed